tensorflow_quantum/core/src/circuit_parser_qsim.cc

```diff
@@ -641,6 +641,25 @@ inline Status DepolarizingChannel(const Operation& op,
   return Status::OK();
 }
 
+inline Status GADChannel(const Operation& op, const unsigned int num_qubits,
+                         const unsigned int time, NoisyQsimCircuit* ncircuit) {
+  int q;
+  bool unused;
+  float p, gamma;
+  Status u;
+  unused = absl::SimpleAtoi(op.qubits(0).id(), &q);
+
+  u = ParseProtoArg(op, "p", {}, &p);
```

Comment:
Should have two separate statuses here, currently bad status from parsing p will be ignored

Reply:
Good catch. Made consistent with other gates.